lib/helpers/blueprint-helper.js

```diff
@@ -177,6 +170,18 @@ function resultContents(options, env, result) {
     outputFiles: env && env.tmpdir && walkSync(env.tmpdir) || null
   };
 }
+function throwResult(options, err) {
+    if (options && options.throws && err && err.statusCode) {
+        if (err.errorLog.length) {
+            // assert the error type if we've defined one to check
+            if (options.throws.type) {
+                expect(err.errorLog[0].name).to.equal(options.throws.type);
+            }
+            throw err.errorLog[0];
+        }
+    }
+    throw err;
+}
```

Comment:
Doh, was working on my work machine and the .editorconfig wasn't being respected